demos/hvl_virtio.rst

```diff
@@ -0,0 +1,87 @@
+=============================
+OpenAMP Hypervisorless Virtio
+=============================
+
+.. _hvl-virtio-intro:
+
+***************************
+Hypervisorless Virtio Intro
+***************************
+
+There are a number of `hypervisorless virtio demos <https://github.com/OpenAMP/zephyr-openamp-staging/tree/virtio-exp/samples/virtio>`_. Their intent is to demonstrate the use of the :ref:`Virtio<overview-rpmsg-work-label>` component and associated drivers.
+
+This section details the sample for `hypervisorless virtio with entropy and network devices (hvl_net_rng_reloc) <https://github.com/OpenAMP/zephyr-openamp-staging/tree/virtio-exp/samples/virtio/hvl_net_rng_reloc>`_.
+
+All demonstrations are run on a remote running `Zephyr Operating System (OS) <https://www.zephyrproject.org/>`_.
+
+..  image::  ../images/demos/hvl-virtio-intro.svg
+
+.. _hvl-virtio-components:
+
+********************************
+Hypervisorless Virtio Components
+********************************
+
+This demonstration uses a single application on the remote and a setup and demo script on the host.
+
+The underlying OpenAMP architectural components used by these applications are
+
+* :ref:`Remoteproc<overview-remoteproc-work-label>`
+* :ref:`Resource Table<overview-remoteproc-work-label>`
+* :ref:`Virtio<overview-rpmsg-work-label>`
+* :ref:`Libmetal<overview-proxy-libmetal-label>`
+
+The following architecture diagram shows the components involved in the demonstration.
+
+..  image::  ../images/demos/hvl-virtio-components.svg
+
+The top-level control flow is shown in the following message diagram.
+
+.. _hvl-virtio-control-flow:
+
+..  image::  ../images/demos/hvl-virtio-control-flow.svg
```

Comment:
The control flow diagram should be updated to remove the "Setup Resource Table" action.